packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -181,9 +137,7 @@ export enum ChainId {
   ArbitrumGoerli = 421613,
   ArbitrumSepolia = 421614,
   ArbitrumLocal = 412346,
-  // Orbit Testnets
-  XaiTestnet = 47279324479,
-  Xai = 660279,
+  // Orbit
   StylusTestnet = 23011913
```

Comment:
shouldn't we remove stylus testnet too if we're removing xai?

Reply:
Stylus Testnet is part of the SDK so we can keep its chain ID